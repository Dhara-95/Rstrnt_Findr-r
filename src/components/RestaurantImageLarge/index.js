import React from "react";

import { checkInvalidImageURL } from "../../utils";

const RestaurantImageLarge = (props) => {
  const { url, name } = props;
  const src = checkInvalidImageURL(url);

  return (
    <div className="flex w-48 h-48 rounded-3xl overflow-hidden relative justify-center content-center ">
      <img
        className="w-auto h-full block object-cover transform scale-130"
        src={src}
        alt={name}
      />
    </div>
  );
};

export { RestaurantImageLarge };
