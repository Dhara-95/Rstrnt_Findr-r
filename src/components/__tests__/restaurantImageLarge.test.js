import React from "react";
import renderer from "react-test-renderer";

import { RestaurantImageLarge } from "../RestaurantImageLarge";

it("displays the large Restaurant image correctly", () => {
  const tree = renderer
    .create(<RestaurantImageLarge url={"url"} name={"name"} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
