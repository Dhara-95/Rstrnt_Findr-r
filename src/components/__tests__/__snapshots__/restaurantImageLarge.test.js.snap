// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the large Restaurant image correctly 1`] = `
<div
  className="flex mx-auto w-full h-80 py-8 overflow-hidden relative justify-center content-center lg:w-2/3 xl:w-2/3 2xl:w-2/3 "
>
  <img
    alt="name"
    className="w-full h-auto block object-cover"
    src="url"
  />
</div>
`;
